### DISCUSSION TITLE
Can i hire a developer here for a wasm nft project, or somthing built on sologenic

### DISCUSSION THREAD

**Discussion by mrkdco** [gnolang/gno#155] [Q&A] [answered]
Trying to build a nft staking staking platform. i have bots that generate income and for people that just hold certain nfts or stake them on our site certain yields willl apply. would like it all automated by smart contract

**Comment by faddat** [comment]
build it on gno

if you gno, you gno

gno doesn't compile to wasm, and so it is (theoretically for now) much more efficient. 

Baiscally here is how a wasmvm setup works in cosmos currently:

* chain (go) -> wasmd (go) -> wasmvm (go + precompiled rust lib accessed through cgo ffi) 

gno skips the wasm part and will look like:

* chain (go) -> contracts (gno)

**Comment by mrkdco** [comment]
I'd like that... can I get some help with that?

I'm even down to start learning it.

**Comment by moul** [comment, marked as answer]
For hiring needs, please use the official Discord right now.
